blog/2023-07-28-july-releases.md

```diff
@@ -0,0 +1,124 @@
+* [`conda.models.leased_path_entry.LeasedPathEntry`](https://github.com/conda/conda/pull/12735)
+* [`conda.models.enums.LeasedPathType`](https://github.com/conda/conda/pull/12735)
+* [`conda.common.temporary_content_in_file`](https://github.com/conda/conda/pull/12795) (use `tempfile` instead)
+* [`conda.cli.python_api`](https://github.com/conda/conda/pull/12796) (use `conda.testing.conda_cli` fixture instead)
+
+
+* * *
+
+## [Changes in Conda-Build](https://github.com/conda/conda-build/releases/tag/3.26.0)
+
+To update conda-build, run:
+```bash
+conda install -n base conda-build=3.26.0
+```
+
+### ✨ What's New? ✨
+
+In the latest version of conda-build, extra-meta data is logged to make it easier to verify that the right extra-meta data is burned into packages (which also helps to co-relate packages and their build-log). The feature was first introduced in [PR #4303](https://github.com/conda/conda-build/pull/4303) and is now improved via the logging call.
+
+Additionally, the following features and changes can be found in conda-build 3.26.0:
+
+* `pip` has been added to the `env-doc make` command so that the function works correctly (`pip` is no longer added by default with the python conda package).
+* Subcommands are implemented as conda plugins.
+* Duplicate `get_summary` calls are dropped in `conda_build.skeletons.pypi`.
+* `resolved_packages` tests that failed due to recent OpenSSL 3.0.8 release to defaults is now fixed.
+
+
+### 🔧 What Got Fixed? 🔧
+
+The following bug fixes were implemented in the 3.26.0 version of conda-build:
+
+* The handling of unknown binaries with newer `(py)lief` versions has been fixed.
+* `LIEF` logging to remove "Unknown format" warning message is disabled.
+* `enable_static` default value in `conda_build.config` to remove "Failed to get_static_lib_exports" warning messages has been reverted.
+* We now avoid duplicate logging by not propagating the top-level conda-build logger.
+* Git cloning for repositories with submodules containing local relative paths has been fixed.
+
+
+### 📄 What's New in Documentation? 📄
+
+* `pkg-spec` docs have been updated to mention `.conda` package format.
+* Unnecessary Jinja package name variables from `variants.rst` docs file have been dropped.
+
+
+### 🌅 What's Marked for [Deprecation](https://github.com/conda-incubator/ceps/blob/main/cep-9.md)? 🌅
+
+* [Executable invocations (e.g., `conda-build`)](https://github.com/conda/conda-build/pull/4921)
+* [Module-based invocations (e.g., `python -m conda_build.cli.main_build`)](https://github.com/conda/conda-build/pull/4921)
+
+
+* * * 
+
+### We ❤️ Our Community
+
+Altogether we had 10 (!) new contributors to the conda and conda-build repositories this release cycle; thank you to all of our open source community members for helping make the new versions of conda and conda-build so great.
```

Comment:
I know it's tedious, but would it be worth the effort add a list of new contributors, and links to their GitHub pages?

I can submit a PR if the this is worthwhile, but the thought of creating it makes you shake.

Reply:
@tnabtaf Just add the suggestion here, don't create another PR, this gets harder to review that way

![CleanShot 2023-07-27 at 11 27 06@2x](https://github.com/conda-incubator/conda-dot-org/assets/1610/fa299ee8-f5c5-4cb3-838b-1c561e90dda3)

Reply:
```suggestion
Altogether we had 10 (!) new contributors to the conda and conda-build repositories this release cycle; thank you to all of our open source community members for helping make the new versions of conda and conda-build so great.

* [@johanneskoester](https://github.com/johanneskoester) made their first contribution in [conda#12683](https://github.com/conda/conda/pull/12683)
* [@josegonzalez](https://github.com/josegonzalez) made their first contribution in [conda-build#3998](https://github.com/conda/conda-build/pull/3998)
* [@kalawac](https://github.com/kalawac) made their first contribution in [conda#12738](https://github.com/conda/conda/pull/12738)
* [@lesteve](https://github.com/lesteve) made their first contribution in [conda#12715](https://github.com/conda/conda/pull/12715)
* [@marcoesters](https://github.com/marcoesters) made their first contribution in [conda#12863](https://github.com/conda/conda/pull/12863)
* [@mattkram](https://github.com/mattkram) made their first contribution in [conda#12730](https://github.com/conda/conda/pull/12730)
* [@morremeyer](https://github.com/morremeyer) made their first contribution in [conda#12871](https://github.com/conda/conda/pull/12871)
* [@mpotane](https://github.com/mpotane) made their first contribution in [conda#11740](https://github.com/conda/conda/pull/11740)
* [@rfezzani](https://github.com/rfezzani) made their first contribution in [conda-build#4850](https://github.com/conda/conda-build/pull/4850)
* [@schackartk](https://github.com/schackartk) made their first contribution in [conda#12781](https://github.com/conda/conda/pull/12781)
```